packages/frame-host/src/iframe.ts

```diff
@@ -21,11 +21,11 @@ export function createIframeEndpoint({
     ...Comlink.windowEndpoint(iframe.contentWindow!),
     emit: (event) => {
       if (debug) {
-        console.debug("frameEvent", event);
+        console.debug("FarcasterFrameEvent", event);
       }
 
       const wireEvent = {
-        type: "frameEvent",
+        type: "FarcasterFrameEvent",
```

Comment:
this won't be backwards compatible?

Reply:
the old one is broken because we never used this path for sending, instead sent directly to the react reference

Reply:
but yeah we could keep those and never unify the types between react native and web